### PR TITLE
For Fenix#7517: Update menu container style to use 16dp padding

### PR DESCRIPTION
**Pull request by psymoon**
Also allow `mozac_browser_menu_web_extension` to use `@style/Mozac.Browser.Menu.Item.Container`.

### CODE REVIEW

File: components/browser/menu/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -11,8 +11,8 @@
     <!--Menu Item -->
     <dimen name="mozac_browser_menu_item_text_size">16sp</dimen>
     <dimen name="mozac_browser_menu_item_container_layout_height">48dp</dimen>
-    <dimen name="mozac_browser_menu_item_container_padding_start">24dp</dimen>
```
Comment (Amejia481) [inline components/browser/menu/src/main/res/values/dimens.xml]: Is it there any specific reason why do we want to change padding for all the menu items?

Comment (psymoon) [reply components/browser/menu/src/main/res/values/dimens.xml]: @Amejia481 https://github.com/mozilla-mobile/fenix/issues/7157

Comment (Amejia481) [reply components/browser/menu/src/main/res/values/dimens.xml]: Thanks, I just saw the change in isolation web ext!

**Comment by psymoon** [comment]
bors r=Amejia481

**Pull request merged**